pybandits/strategy.py

```diff
@@ -221,11 +221,14 @@ def select_action(self, p: Dict[ActionId, Probability], actions: Dict[ActionId,
         # define the set of feasible actions
         feasible_actions = [a for a in p.keys() if p[a] >= (1 - self.subsidy_factor) * max_p]
 
-        # feasible action costs
-        _, cheapest_action = min((b.cost, a) for a, b in actions.items() if a in feasible_actions)
+        # feasible actions enriched with their characteristics (cost, 1-probability, action_id)
+        sortable_actions = [(actions[a].cost, 1 - p[a], a) for a in feasible_actions]
+
+        # best action is the one with cheapest cost (+ higher proba in case of equality)
```

Comment:
I guess we can also modify the comment here:
`# select the action with the cheapest cost (and the highest probability in case of cost equality)`

Reply:
Done! thanks